kernel/kernel-api/src/main/java/io/delta/kernel/internal/icebergcompat/IcebergWriterCompatV1MetadataValidatorAndUpdater.java

```diff
@@ -66,6 +67,30 @@
 public class IcebergWriterCompatV1MetadataValidatorAndUpdater
     extends IcebergCompatMetadataValidatorAndUpdater {
 
+  /**
+   * Validates that any change to property {@link TableConfig#ICEBERG_WRITER_COMPAT_V1_ENABLED} is
+   * valid. Currently, the changes we support are
+   *
+   * <ul>
+   *   <li>No change in enablement (true -> true or false -> false)
```

Comment:
Might be worth commenting exactly what you do _not_ support, which is enablement on an existing table?

Q: what's the value of oldConfig on an new table? Just empty map? Just curious.

Reply:
Clarified a bit in the last line -- what we don't support.

> Q: what's the value of oldConfig on an new table? Just empty map? Just curious.

Correct